lib/host/ui/register_ao_uis.js

```diff
@@ -10,7 +10,7 @@ const { PLATFORM = 'bitfinex' } = process.env
  *
  * @param {Object} aoHost - algo order host
  */
-module.exports = async (aoHost = {}) => {
+module.exports = async (aoHost = {}, label = null) => {
```

Comment:
What is label here?

Reply:
label of a AO: 'Ping/pong', 'Iceberg' etc.

Reply:
Right got you, I dont this is the best solution either as this means we can only ever disable one order at a time  and it also creates a bit of a confusing side effect. Leave this with me, im going to do a spike to see if we can add this logic to the hf-server instead, Im thinking we could totally destroy and re-create the hf-algo instance when a user re-registers. This way, not only will it be re-registered from the UI but it also wont be able to perform any updates